sdk/proto/phpfpm.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package f5.nginx.agent.sdk;
+
+import "gogo.proto";
+
+option go_package = "github.com/nginx/agent/sdk/v2/proto;proto";
+
+// PhpFpmDetails registers a phpfpm master process + child pools
+message PhpFpmDetails {
+  // PhpFpm process type (e.g. phpfpm)
+  string type = 1 [(gogoproto.jsontag) = "type"];
+  // System ID
+  string uuid = 2 [(gogoproto.jsontag) = "uuid"];
+  // PhpFpm ID
+  string local_id = 3 [(gogoproto.jsontag) = "local_id"];
```

Comment:
we've moved away from the local_id convention, perhaps this should be called php_id or server_id?